PoS review fixes (#677)
0xPolygon/polygon-docs: docs/pos/get-started/building-on-polygon.md

comments: true
---

## Overview

All your favorite Ethereum tools (Foundry, Remix, Web3.js) work seamlessly on Polygon, with the same familiar UX. Just switch to the [Polygon RPC](https://polygon-rpc.com/) and keep building.

Connect your wallet and deploy any decentralized application to either PoS mainnet or Amoy testnet (Sepolia-anchored).

Use the links below to find the right tooling and guides that suit your needs the best.

- [Faucets](../../tools/gas/matic-faucet.md) - Fetch test tokens
- [Polygon gas station](../../tools/gas/polygon-gas-station.md) - Gas estimation API
- [Polygon dApp Launchpad](../../tools/dApp-development/launchpad/intro.md) - dApp development CLI tool
- [Popular third-party tooling](../../tools/dApp-development/third-party-tutorials.md)
- [Matic.js library](../../tools/matic-js/installation.md)

> Decentralized applications (dApps) act as the bridge between users and their data privacy on the blockchain. The increasing number of dApps validates their usefulness within the blockchain ecosystem, solving challenges like executing transactions between two participants without the need for central authority via smart contracts.

Reviewer: 1. Does the first sentence refer to all or a particular set of dApps? Is there a discussion on how this actually achieved? 
2. If the word "bridge" here means the same as "link", I would suggest you use "link". However, I must admit that the idea of 'linking users to their data privacy' sounds a bit off the mark.  
3. I get a sense that the message could be that "these dApps somehow guarantee privacy of users' (personal) data." ... but hence the need to qualify which dApps are being referred to here.

Please consider revising the sentence.

Author: Yeah that isn't a good sentence. Brought it in from another spot. I wanted to replace it with something else. It can be deleted though. Not really necessary.

Reviewer: On the second sentence:

- I am trying to understand how "The increasing number of dApps validates their usefulness"? This is not clear to me.
- I am not sure that p2p txs are possible without using at least one smart contract (even if it's within one chain).   

The trouble here (with line 19) is that it is commentary in the middle of a guide, and thus warrants to be glaringly relevant to what the user needs to do. This one does not achieve this goal. 
So the question is: Can the user do without this commentary?

Author: Yes. I just updated that bit with a new commit.

main();
```

!!! tip "Reach out to us!"

>     Run into issues while hacking? Encounter something that you're having a hard time understanding? Use the following methods to reach out to us:

Reviewer: ```suggestion
    Running into issues while hacking? Encountering something hard to understand? Use the following methods to reach out to us:
```
